tsl/test/sql/compression_errors.sql

```diff
@@ -100,7 +101,7 @@ ALTER TABLE foo set (timescaledb.compress, timescaledb.compress_segmentby = 'b,
 ALTER TABLE foo set (timescaledb.compress, timescaledb.compress_orderby = 'b, b');
 
 --should succeed
-ALTER TABLE foo set (timescaledb.compress, timescaledb.compress_orderby = 'a, b');
+ALTER TABLE foo set (timescaledb.compress, timescaledb.compress_orderby = 'a, b', timescaledb.compress_segmentby='');
```

Comment:
Why did you have to add this? IIRC if you specify orderby and don't specify segmentby, it implies null segmentby. Does this still work?